src/braket/default_simulator/state_vector_simulator.py

```diff
@@ -127,12 +139,92 @@ def properties(self) -> GateModelSimulatorDeviceCapabilities:
                             {"name": "DensityMatrix", "minShots": 0, "maxShots": 0},
                             {"name": "Amplitude", "minShots": 0, "maxShots": 0},
                         ],
-                    }
+                    },
+                    "braket.ir.openqasm.program": {
+                        "actionType": "braket.ir.openqasm.program",
+                        "version": ["1"],
+                        "supportedOperations": [
+                            # OpenQASM primitives
+                            "U",
+                            "GPhase",
```

Comment:
Is this also true for JAQCD? I'm not sure `properties` actually needs to be implemented at this base level.

Reply:
No, that's why it's only listed under `braket.ir.openqasm.program` and not `braket.ir.jaqcd.program`. Because of the class structure, `StateVectorSimulator` is a subclass to the IR-specific simulators and not the other way around. So this isn't a "base level", it's actually the sub-est class. (more on this in response to your diamond structure comment)